wirecard/prestashop-ee
# Fallback language

### This PR

* Default language fallback for missing languages

### Notes

* Not mergeable yet

### TODO

* Before this branch can be merged, the Phraseapp integration branch has to be merged
* In the script translation-builder.rb, line 81 needs to be changed 
from: translation_keys = file.read.scan(/\{l s=\'(.*)\' mod/).uniq
to: translation_keys = file.read.scan(/\{lFallback s=\'(.*)\' mod/).uniq

### How to test

* switch between different languages in the shop and administration section and check translations
-> no key should be visible
-> for missing translations the English string should be shown
